docs/admin/admin-ui-setup/admin-ui-users.md

```diff
@@ -24,10 +32,41 @@ To create a new user:
 4. Review the new user information and note the temporary password that has been assigned. To send the user an introductory email, select the checkbox.
 5. Press *Done* when complete.
 
+## Users Table
+
+The *Users* table displays a list of users who are authorized to login to the Run:ai platform.
+
+!!! Note
+    SSO users who have not logged into the Run:ai platform will not appear in the table until they do.
+
+The *Users* table contains informtion about users who have logged into the Run:ai Platform. The table includes columns for:
+
+* **User**&mdash;the email and user ID of the user.
+* **Type**&mdash;the type of user (local or SSO).
+* **Last login**&mdash;the last time the user logged into the platform.
+* **Access rule(s)**&mdash;press *VIEW* to view the access rule(s) assigned to the user.
+* **Created by**&mdash;indicates who created the user in the able.
+* **Creation time**&mdash;the timestamp for when the user was created.
+* **Last updated**&mdash;the last time the user information was updated.
+
+When you press *View* in the *Access rule(s)* column, a pop-up will appear that displays the rules assigned to the user. In the popup are the following columns:
+
+* **Role**&mdash;the name of the role assigned to the user.
+* **Scope**&mdash;the scope to which the user has rights.
+* **Type**&mdash;the type of user (SSO or other)
+* **Group**&mdash;the group to which the user is assigned. (Applies only to SSO users)
```

Comment:
@jasonnovichRunAI 
```suggestion
* **Group**&mdash;if the access rule is inherited from an SSO group, the name of the group assigned to the access rule (Applies only to SSO users).
```

Reply:
```suggestion
* **Group**&mdash;the group to which the user is assigned. If the access rule is inherited from an SSO group, the name of the group assigned to the access rule (Applies only to SSO users).
```
@gal-revach WDYT?

Reply:
@jasonnovichRunAI we still don't need the first sentence "the group to which the user is assigned" because it is inaccurate and confusing... maybe we should write "the name of the group assigned to the access rule, in case the access rule is inherited from an SSO group (applies only to SSO users)."
WDYT?